p2p/switch.go

```diff
@@ -491,11 +492,14 @@ func (sw *Switch) listenerRoutine(l Listener) {
 			break
 		}
 
-		// ignore connection if we already have enough
-		// leave room for MinNumOutboundPeers
-		maxPeers := sw.config.MaxNumPeers - DefaultMinNumOutboundPeers
```

Comment:
Should we have some bias towards accepting incoming peers depending on our current inbound / outbound peer ratio? Thats something we can do postlaunch though.

Reply:
I thought about it a little bit. We can give greater control by introducing MaxNumInboundPeer/MaxNumOutboundPeers or InOutPeerRatio or something else. But it's perfectly fine to do it later.

Reply:
Sounds good to me! Should we make an issue for that?